luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -409,8 +413,35 @@ public Builder(LuthierResourceDictionaries resourceDictionaries, GranularityDict
             return logicalTableFactoryMap;
         }
 
-        public GranularityDictionary getGranularityDictionary() {
-            return granularityDictionary;
+        /**
+         * Default LogicalTable factories that are defined in fili-core.
+         *
+         * @return  a LinkedHashMap of MetricMaker name to its factory
```

Comment:
The fact that it's a `LinkedHashMap` doesn't really matter (if it does, we should make that its return type). Better to just say "Map."

Reply:
Got it.